.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -7,6 +7,25 @@ labels: kind/bug
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 
+**Environment**:
```

Comment:
How about keep the `Environment` at the bottom

Reply:
ok